pkgs/client/src/lib/FlowStep.ts

```
@@ -0,0 +1,258 @@
+          status: FlowStepStatus.Failed,
+          failed_at: typeof event.failed_at === 'string' ? new Date(event.failed_at) : new Date(),
+          error_message: typeof event.error_message === 'string' ? event.error_message : 'Unknown error',
+          error: new Error(typeof event.error_message === 'string' ? event.error_message : 'Unknown error'),
+        };
+        this.#events.emit('failed', event);
+        break;
+
+      default:
+        // Exhaustiveness check - should never happen with proper types
+        // @ts-expect-error Intentional exhaustiveness check
+        const _exhaustivenessCheck: never = event;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)